elements/pfe-navigation/src/pfe-navigation.scss

```diff
@@ -72,6 +72,24 @@ $variables: map-merge($variables, $nav-variables);
   z-index: #{pfe-zindex(navigation)};
 }
 
+:host([pfe-full-width]) {
+  .pfe-navigation {
+    &__container {
+      @each $size in ("md", "lg", "xl") {
```

Comment:
```
:host([pfe-full-width]) .pfe-navigation__container {
  @each $size in ("md", "lg", "xl") {
    @media screen and (min-width: pfe-breakpoint( #{$size} ) ) {
      @if ($size == "md" or $size == "lg") {
        --pfe-navigation--Width: calc(100% - #{pfe-local(Padding--horizontal)} * 4);
      } @else if ($size == "xl") {
        --pfe-navigation--Width: calc(100% - #{pfe-local(Padding--horizontal)} * 8);
      }
    }
  }
}
```

Reply:
^ This would simplify the logic here a bit

Reply:
@castastrophe Thanks. I just updated the code with your suggestion.